feconf.py

```diff
@@ -295,6 +295,12 @@
 
 
 def get_empty_ratings():
+    """Returns a copy of the Empty Ratings object.
+
+    Returns:
+        dict: Copy of the '_EMPTY_RATINGS' dict object which contains the empty
```

Comment:
Please replace colon(:) to- `dict. Copy of the '_EMPTY_RATINGS' dict object which contains the empty ratings.` for consistency.